### PR TITLE
IA-2494 Change Requests API feedback

### PR DESCRIPTION
**Pull request by kemar**
Change Requests API feedback.

Related JIRA tickets : [IA-2494](https://bluesquare.atlassian.net/browse/IA-2494)

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [x] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- Allow to create an `OrgUnitChangeRequest` using an `OrgUnit`'s `ìd` or `uuid` (via a custom `IdOrUuidRelatedField`)

[IA-2494]: https://bluesquare.atlassian.net/browse/IA-2494?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: iaso/api/org_unit_change_requests/serializers.py [hunk 1]
```diff
@@ -177,7 +178,7 @@ class OrgUnitChangeRequestWriteSerializer(serializers.ModelSerializer):
     """
 
     uuid = serializers.UUIDField(required=False, default=uuid.uuid4)
-    org_unit_id = serializers.PrimaryKeyRelatedField(
+    org_unit_id = IdOrUuidRelatedField(
         source="org_unit",
         queryset=OrgUnit.objects.all(),
         required=False,
```
Comment (bmonjoie) [inline iaso/api/org_unit_change_requests/serializers.py:184]: It is required. Otherwise, to which OrgUnit do you assign this change request?

**Review by bmonjoie — APPROVED** [review]
It looks nice and clean, good job :+1:

File: iaso/api/org_unit_change_requests/serializers.py (hunk 1 above)
Comment (kemar) [reply iaso/api/org_unit_change_requests/serializers.py:184]: It's used for both `POST` and `PUT` and `org_unit_id` isn't required on update.
But we can make it required, WDYT?

Comment (bmonjoie) [reply iaso/api/org_unit_change_requests/serializers.py:184]: It's fine as is if it is required on `POST` :slightly_smiling_face:

**Pull request merged**